### PR TITLE
Feature/rw 404 and stats

### LINKED ISSUE #48: Under Construction / Does Not Exist
### Feature Categories:
- React

**Describe the feature/Bug your gonna work on**
- Add an Under Construction & Does Not Exist component

**Additional context**
- After testing out our Route implementation, we realized it would create a more seamless user experience if we could route them to error pages.

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- PageNotFound component added and tested for any 404 path 
- Stats component added and tested - currently it is an under construction page 
- Moved Provider components to wrap around App component so that Router works correctly 

**Note: We may want to look into adding useMemo for our components using context to avoid unnecessary re-renders**

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration: `npm test` 

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

resolve #48

### CODE REVIEW

File: src/Components/About/About.module.scss [hunk 1]
```diff
@@ -25,4 +25,16 @@
...
+  .appDescription {
+    margin: 2em 8em; 
+  }
+}
+
+@media screen and (max-width: 688px) {
+  .appDescription {
+    margin: 2em 2em; 
+  }
 }
```
Comment (Abdeboskey) [inline src/Components/About/About.module.scss:40]: Responsiveness YESSS

File: src/index.js [hunk 2]
```diff
@@ -2,12 +2,18 @@ import React from 'react';
...
   <Router>
-    <App />
+    <SettingsProvider>
+      <ViewProvider>
+        <App />
+      </ViewProvider>
+    </SettingsProvider>
   </Router>,
   document.getElementById('root')
 );
```
Comment (Abdeboskey) [inline src/index.js:19]: for ease of understanding, I really like the providers here.

File: src/Components/PageNotFound/PageNotFound.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import style from './PageNotFound.module.scss'
+import burnout from '../../assets/burnout3.jpg'
+
+const PageNotFound = () => {
+  return (
+    <section>
+      <h2>Oops, we can't seem to find the page you're looking for!</h2>
+      <img className={style.burnout} src={burnout} alt='distressed internet user'/>
```
Comment (Abdeboskey) [inline src/Components/PageNotFound/PageNotFound.js:9]: Love this image, great job fren!

File: src/Components/Stats/Stats.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import style from './Stats.module.scss'
+
+const Stats = () => {
+  return (
+    <section>
+      <h2>Stats Page Coming Soon!</h2>
+      <section className={style.statsDesc}>
+        <p>Check back soon for personal stats so you can see the positive impact Som Timer is having on your productivity and mental health. Stats will include data such as a log of pom cycles taken, mood and engagement tracking, and what types of content work best for you!</p>
+      </section>
```
Comment (Abdeboskey) [inline src/Components/Stats/Stats.js:10]: Can we use an `<article>` tag for this instead?

**Review by Abdeboskey — APPROVED** [review]
Perfect! Thank you for making these!

File: src/Components/PageNotFound/PageNotFound.js (hunk 3 above)
Comment (jkwest-93) [inline src/Components/PageNotFound/PageNotFound.js:9]: I love this visual element to the 404! Great find.

File: src/index.js [hunk 5]
```diff
@@ -2,12 +2,18 @@ import React from 'react';
...
 import App from './Components/App/App';
+import { ViewProvider } from './Context/ViewContext'
+import { SettingsProvider } from './Context/SettingsContext'
 import { BrowserRouter as Router } from 'react-router-dom'
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <Router>
-    <App />
+    <SettingsProvider>
```
Comment (jkwest-93) [inline src/index.js:12]: Too weird that we're getting issues with Provider & Router! Looking forward to learning more about this & `useMemo`.

**Review by jkwest-93 — APPROVED** [review]
This looks great! Glad we're getting ahead of all the ways in which our site can "break"

File: src/Components/About/About.module.scss (hunk 1 above)
Comment (sienna-kopf) [reply src/Components/About/About.module.scss:40]: is that what this does!! SO cool!

File: src/Components/PageNotFound/PageNotFound.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+      <section className={style.notFoundContent}>
+        <p>Here are some helpful links to get you back on track:</p>
```
Comment (sienna-kopf) [inline src/Components/PageNotFound/PageNotFound.js:11]: I love the idea of giving the user paths to return to non-404 land!

File: src/Components/PageNotFound/PageNotFound.test.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    const homeLink = getByRole('link', { name: 'Home / Timer'})
+    const settingsLink = getByRole('link', { name: 'Settings'})
+    const aboutLink = getByRole('link', { name: 'About'})
+
+    expect(notFoundHeading).toBeInTheDocument()
+    expect(notFoundImg).toBeInTheDocument()
+    expect(helpfulLinksText).toBeInTheDocument()
+    expect(homeLink).toBeInTheDocument()
+    expect(settingsLink).toBeInTheDocument()
+    expect(aboutLink).toBeInTheDocument()
```
Comment (sienna-kopf) [inline src/Components/PageNotFound/PageNotFound.test.js:27]: Love this thorough test of the elements!

**Review by sienna-kopf — APPROVED** [review]

File: src/Components/Stats/Stats.js (hunk 4 above)
Comment (rwilliams659) [reply src/Components/Stats/Stats.js:10]: Changed! Good suggestion

**Pull request merged**